README.md

```diff
@@ -140,6 +140,7 @@ Add to your `.babelrc`'s plugins array.
 
 ## Benchmarks
 > Bootstrap: `npm run bootstrap`
+> Build: `npm run build`
```

Comment:
I wonder if you need to make it a little clearer that you need to run the build before you attempt to run the bootstrap? Maybe just -

`Bootstrap (requires build): ...`